src/VirtoCommerce.InventoryModule.Core/VirtoCommerce.InventoryModule.Core.csproj

```diff
@@ -10,6 +10,6 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="VirtoCommerce.CoreModule.Core" Version="3.1.0" />
-    <PackageReference Include="VirtoCommerce.Platform.Core" Version="3.5.0" />
+    <PackageReference Include="VirtoCommerce.Platform.Core" Version="3.32.0" />
```

Comment:
Probably we need to use 3.12 version, where `ILastModifiedDateTime` was introduced, as we are using a minimum required version in dependencies.